src/lib/SlackClient.py

```diff
@@ -54,38 +57,51 @@ def check_command_format(self, command):
 
         message = ''
         response_type = ''
-        is_valid = False
+        is_invalid = False
```

Comment:
y tho

Reply:
The extra argument means there's logic changes in how we look at the command. Its simpler to say that the command is valid and must be proven otherwise, than to assume it's wrong and must be proven valid.